requirements.txt

```diff
@@ -7,6 +7,7 @@ gunicorn[gevent]
 humanize==0.5.1
 prometheus_client
 prometheus-flask-exporter==0.2.0
+pybreaker==0.4.4
```

Comment:
Why did you decide on `pybreaker`? I thought you were going to use `circuitbreaker`? I know very little about this, but just from reading their PyPi pages, `circuitbreaker` looks like it is more mature and has more features.

Reply:
I used this module because `circuitbreaker` looks more mature. But it actually doesn"t work. I played a bit arround with it and some features are not implemented or do not work... This module has more users that uses it and more devs on it.

Reply:
@tbille good answer =)